### PR TITLE
Update(armory.io): Rewording and Version Update

### PR DESCRIPTION
**Pull request by kinnonyee-armory**
First pass to change some rewording and version information

Loosely tied to [CDSH-529]

deploy preview:  https://deploy-preview-2239--armory-docs.netlify.app/continuous-deployment/release-notes/rn-armory-spinnaker/

[CDSH-529]: https://armory.atlassian.net/browse/CDSH-529?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mknous — COMMENTED** [review]
Wonder if we should change "like" to such as for better grammar?

**Review by aimeeu — APPROVED** [review]

**Review by kyledmitchell — APPROVED** [review]

File: content/en/continuous-deployment/release-notes/rn-armory-spinnaker/_index.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ To install, upgrade, and configure Armory Continuous Deployment (Armory CD), ens
...
-| 2.25.x            | 1.2.6                    |
+
 
 ## LTS releases
 
-Starting with version 2.27.1, Armory CD follows a Long Term Support (LTS) release model where there are two LTS releases per year. Major features and changes are packaged into an LTS release.
+Starting with version 2.27.1, Armory CD follows a Long Term Support (LTS) release model with two LTS releases per year. Significant features and changes are packaged into an LTS release.
 
-Each LTS is supported for 1 year from its release date. During the supported lifetime of an LTS release, Armory ships patch releases for the LTS that include improvements like CVE remediation, bug fixes, and performance improvements.
+Each LTS is supported for at least 1 year from its release date. During the supported lifetime of an LTS release, Armory ships patch releases for the LTS that include improvements such as CVE remediation, bug fixes, and performance improvements.
```
Comment (kyledmitchell) [inline content/en/continuous-deployment/release-notes/rn-armory-spinnaker/_index.md]: Change this to: "Armory supports the three (3) most recent major LTS releases. During the supported lifetime of an LTS release, Armory ships minor releases for the LTS that include improvements such as CVE remediation, bug fixes, and performance improvements.  Non-breaking features or enhancements may also be included and will typically be placed behind a feature flag."

File: content/en/continuous-deployment/release-notes/rn-armory-spinnaker/_index.md [hunk 2]
```diff
@@ -23,14 +23,14 @@ To install, upgrade, and configure Armory Continuous Deployment (Armory CD), ens
...
```
Comment (kyledmitchell) [inline content/en/continuous-deployment/release-notes/rn-armory-spinnaker/_index.md:37]: Add a new section below "LTS releases" called "Feature releases" with following text

Starting with version 2.31.0 Armory CD is introducing feature releases which are an additional type of major release that will complement LTS releases.  Feature releases will be released in between the two major LTS releases each year and will have a shortened support window of 6 months.   

Feature releases will 

- Enable Armory customer to test and consume Armory & OSS features faster
- Improve the quality and stability of LTS releases by having changes pre-tested and validated ahead of time within the feature releases

LTS releases will continue to go out roughly every 6 months with one or two feature releases also going out between them.

**Review by kyledmitchell — CHANGES_REQUESTED** [review]

**Comment by kinnonyee-gmail** [comment]
@kyledmitchell 
Your changes have now been added.  The changes should be good to be committed and merged

FYI @aimeeu

**Comment by aimeeu** [comment]
Moved Release Definitions to Feature Status (https://www.notion.so/Release-Docs-Changes-for-2-31-2e32e4a14384418490913d402270e08d?pvs=4) and implemented Kyle's changes:

Move the [[page](https://docs.armory.io/continuous-deployment/release-notes/release-definitions/)](https://docs.armory.io/continuous-deployment/release-notes/release-definitions/) as a child under the Feature Status section in left nav 

Update [nav title] to…
Feature Status Definitions

Update [page title]
Armory Continuous Deployment Feature Status Definitions

Update [sub title/description] to…
Definitions for feature states for Armory Continuous Deployment Self-Hosted or Managed

Update “Overview of the definitions for different release states” —> “Overview”

Update “Overview” description to…

“Features and [[plugins](https://docs.armory.io/plugins/)](https://docs.armory.io/plugins/) that Armory release have a corresponding status indicating the overall maturity of the feature.  The page summarizes those feature status definitions and provides recommendations on where and how they can be used.”

Delete “Experiment” section

Update “Where to install Beta” text
Delete the first sentence “Pretty much anywhere except production” but leave everything else.  So it should start as “A production installation could occur…”

**Review by aimeeu — APPROVED** [review]

**Pull request merged**